cmd/root.go

```diff
@@ -11,13 +11,8 @@ import (
 // rootCmd represents the base command when called without any subcommands
 var rootCmd = &cobra.Command{
 	Use:   "opcap",
-	Short: "A brief description of your application",
-	Long: `A longer description that spans multiple lines and likely contains
-examples and usage of using your application. For example:
-
-Cobra is a CLI library for Go that empowers applications.
-This application is a tool to generate the needed files
-to quickly create a Cobra application.`,
+	Short: "A cli tool to gauge capabilities of operators available on OpenShift's operator hub.",
+	Long:  `Opcap is a cli tool that interacts with all operators available on OpenShift's operator hub in a live cluster, to assess how smart and cloud native they are.`,
```

Comment:
Looks like anytime we refer to the tool as "opcap" we have it uncapitalized, and elsewhere it's called OpenShift Operator Capabilities Tool, so I think this long form should start with something like "The OpenShift Operator Capabilities tool (opcap) is a command line interface tool... etc"

Also I think we can skip the "interact with" bit and just say "The OpenShift Operator Capabilities Tool (opcap) is a command line interface that assesses the capability levels of operators from a specified catalog source"